week-2/F-logical-operators/exercise2.js

```diff
@@ -5,17 +5,32 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+ var res = num < 0;
+ return res; 
+}
+function isBetween5and10(num){
+var res = 5 < num < 10;
+return res
+}
+function isLongName(name){
+ var res = name.length  < 5;
```

Comment:
Interesting. So if the name is longer than 5 characters, isLongName returns false? That seems counter-intuitive :-)